src/main/java/net/nuagenetworks/bambou/ssl/DynamicKeystoreGenerator.java

```diff
@@ -0,0 +1,156 @@
+package net.nuagenetworks.bambou.ssl;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.KeyFactory;
+import java.security.KeyManagementException;
+import java.security.KeyStore;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.KeyStore.PrivateKeyEntry;
+import java.security.KeyStore.ProtectionParameter;
+import java.security.KeyStoreException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.interfaces.RSAPrivateKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.KeyManagerFactory;
+
+import org.apache.commons.codec.binary.Base64;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DynamicKeystoreGenerator {
+    private static final Logger logger = LoggerFactory.getLogger(DynamicKeystoreGenerator.class);
+
+    private static final char[] KEY_PASSWORD = "changeit".toCharArray();
+	public static final String KEYSTORE_ALIAS = "somealias";
+	public static final String KEY_FACTORY_INSTANCE_TYPE = "RSA";
+	private static final String KEY_MANAGER_FACTORY_ALGORITHM = "SunX509";
+	private static final String KEYSTORE_INSTANCE_TYPE = "JKS";
+	private static final String CERTIFICATE_FACTORY_INSTANCE_TYPE = "X.509";
+	private static final String KEY_CHARSET = "UTF-8";
+	private static final String PRIVATE_KEY_HEADER_TRAILER_REGEX = "(-+BEGIN PRIVATE KEY-+\\r?\\n|-+END PRIVATE KEY-+\\r?\\n?)";
+    
+    private DynamicKeystoreGenerator() {}
+	
+	public static KeyManager[] generateKeyManagersForCertificates(String[] certificateFilePairPaths) throws KeyManagementException {
+        logger.debug("Creating a key manager for files : " + certificateFilePairPaths[0] + " and " + certificateFilePairPaths[1]);
```

Comment:
Indentation issue.  Make sure to use spaces instead of tabs. Applies to all changes.